src/collective/importexport/browser/import_view.py

```diff
@@ -9,7 +10,22 @@
 from zope.annotation import IAnnotations
 from zope.globalrequest import getRequest
 from zope.schema import getFieldsInOrder
-from zope.schema.interfaces import IContextSourceBinder, IBool, IText, IBytes, IInt, IFloat, IDecimal, IChoice, IDate, IDatetime, ITime
+from zope.schema.interfaces import (
```

Comment:
If you do multi-line imports, make sure to use proper spacing. Though Plone style guide [recommends](http://docs.plone.org/develop/styleguide/python.html#about-imports) one-line imports so it's easier to sort them.